### PR TITLE
Fixed issue with duplicated headers in requests.

### PR DESCRIPTION
**Pull request by TPetukhova**
+ Fixed similar issues for query params, cookies.

### CODE REVIEW

**Comment by ElenaPoduzova** [comment]
Review in progress.

File: jdi-dark/src/main/java/com/epam/http/requests/RestMethod.java [hunk 1]
```diff
@@ -277,16 +280,23 @@ public RequestSpecification getSpec() {
...
         if (data.queryParams.any()) {
             spec.queryParams(data.queryParams.toMap());
             data.url += "?" + print(data.queryParams.toMap(), "&", "{0}={1}");
         }
         if (data.body != null)
             spec.body(data.body);
         if (data.headers.any())
-            spec.headers(data.headers.toMap());
+            ((FilterableRequestSpecification) spec).removeHeaders();
+        spec.headers(data.headers.toMap());
```
Comment (ElenaPoduzova) [inline jdi-dark/src/main/java/com/epam/http/requests/RestMethod.java]: spec.headers(data.headers.toMap()); is out of if (data.headers.any()) verification.

File: jdi-dark/src/main/java/com/epam/http/requests/RestMethod.java [hunk 2]
```diff
@@ -277,16 +280,23 @@ public RequestSpecification getSpec() {
...
         if (data.cookies.any())
-            spec.cookies(data.cookies.toMap());
+            ((FilterableRequestSpecification) spec).removeCookies();
+        spec.cookies(data.cookies.toMap());
```
Comment (ElenaPoduzova) [inline jdi-dark/src/main/java/com/epam/http/requests/RestMethod.java]: spec.cookies(data.cookies.toMap()); is out of if (data.cookies.any()) verification.

File: jdi-dark/src/main/java/com/epam/http/requests/RestMethod.java (hunk 1 above)
Comment (TPetukhova) [reply jdi-dark/src/main/java/com/epam/http/requests/RestMethod.java]: fixed, thanks

File: jdi-dark/src/main/java/com/epam/http/requests/RestMethod.java (hunk 2 above)
Comment (TPetukhova) [reply jdi-dark/src/main/java/com/epam/http/requests/RestMethod.java]: fixed, thanks

**Comment by ElenaPoduzova** [comment]
Code review has been done.

**Pull request merged**